frontend/amundsen_application/static/.betterer.results

```diff
@@ -34,7 +34,7 @@ exports[`eslint`] = {
       [163, 6, 29, "Use object destructuring.", "2645724888"],
       [248, 6, 29, "Use object destructuring.", "2645724888"]
     ],
-    "js/components/Bookmark/MyBookmarks/index.tsx:1062745683": [
+    "js/components/Bookmark/MyBookmarks/index.tsx:838732531": [
```

Comment:
Changes in this file is autogenerated with `npm run betterer` in order to pass frontend tests.